java/core/src/main/java/com/suse/manager/webui/services/impl/MonitoringService.java

```diff
@@ -118,8 +119,10 @@ public void setMessages(Map<String, String> messagesIn) {
             if (process.exitValue() != 0) {
                 LOG.error("{} returned non zero exit code: {}", process.info().commandLine(), process.exitValue());
                 try (InputStream stderrIn = process.getErrorStream(); InputStream stdoutIn = process.getInputStream()) {
-                    LOG.error("stderr:\n {}", IOUtils.toString(stderrIn));
-                    LOG.error("stdout:\n{}", IOUtils.toString(stdoutIn));
+                    if (LOG.isErrorEnabled()) {
+                        LOG.error("stderr:\n {}", IOUtils.toString(stderrIn, Charset.defaultCharset()));
+                        LOG.error("stdout:\n{}", IOUtils.toString(stdoutIn, Charset.defaultCharset()));
```

Comment:
You can use modern java also here:
```suggestion
                        LOG.error("stderr:\n{}", new String(stderrIn.readAllBytes(), StandardCharsets.UTF_8));
                        LOG.error("stdout:\n{}", new String(stdoutIn.readAllBytes(), StandardCharsets.UTF_8));
```